docs/developer-docs/integrations/sns/launching/launch-summary.md

```diff
@@ -0,0 +1,532 @@
+
+What we have at this stage:
+
+#### Table 1
+
+<table>
+  <tr>
+    <th>SNS Process</th>
+    <th>State</th>
+  </tr>
+  <tr>
+    <td>Decentralization swap state</td>
+    <td>Pending</td>
+  </tr>
+   <tr>
+    <td>NNS Proposal #1</td>
+    <td>Pending</td>
+  </tr>
+   <tr>
+    <td>SNS Proposal #1</td>
+    <td>Pending</td>
+  </tr>
+   <tr>
+    <td>NNS Proposal #2</td>
+    <td>Pending</td>
+  </tr>
+</table>
+
+#### Table 2
+
+<table>
+  <tr>
+    <th>Objects in an app subnet</th>
+    <th>State</th>
+    <th>Controlled by</th>
+  </tr>
+  <tr>
+    <td>a dapp</td>
+    <td>operational</td>
+    <td>dapp developer principal</td>
+  </tr>
+  <tr>
+    <td class="light-green-text">a principal that can deploy to SNS subnet</td>
+    <td class="light-green-text">pending NNS approval</td>
+    <td class="light-green-text">dapp developer principal</td>
+  </tr>
+</table>
+
+
+### 3. Proposal #1 (of 3) is passed or rejected
+
+This is the **first of three** proposals that need to successfully pass.
+
+If this NNS proposal passes and the developer's principal is added the list of principals that can deploy to the SNS subnet, it does **not** guarantee the rest of the steps will complete.
+
+If the proposal is adopted successfully, at the end of this step, we have:
+
+#### Table 1
+
+<table>
+  <tr>
+    <th>SNS Process</th>
+    <th>State</th>
+  </tr>
+  <tr>
+    <td>Decentralization swap state</td>
+    <td>Pending</td>
+  </tr>
+   <tr>
+    <td>NNS Proposal #1</td>
+    <td class="light-orange-text">Approved</td>
+  </tr>
+   <tr>
+    <td>SNS Proposal #1</td>
+    <td>Pending</td>
+  </tr>
+   <tr>
+    <td>NNS Proposal #2</td>
+    <td>Pending</td>
+  </tr>
+</table>
+
+#### Table 2
+
+<table>
+  <tr>
+    <th>Objects in an app subnet</th>
+    <th>State</th>
+    <th>Controlled by</th>
+  </tr>
+  <tr>
+    <td>a dapp</td>
+    <td>operational</td>
+    <td>dapp developer principal</td>
+  </tr>
+</table>
+
+#### Table 3
+
+<table>
+  <tr>
+    <th>Objects in the SNS subnet</th>
+    <th>State</th>
+    <th>Controlled by</th>
+  </tr>
+  <tr>
+    <td>a principal that can deploy to SNS subnet</td>
+    <td class="light-orange-text">ready for 1-time use</td>
+    <td>dapp developer principal</td>
+  </tr>
+</table>
+  
+
+### 4. Dapp developers trigger the SNS canisters to be created on SNS subnet
+
+When all initial parameters are specified and the NNS approved the SNS launch,
+the SNS canisters can be created by a manual call to [SNS-W](../introduction/sns-architecture.md#SNS-W).
+This will initiate the creation of the SNS canisters and set their initial parameters as
+chosen in [Step 1](#SNS-launch-step-preparation).
+
+**The SNS canisters are created in pre-decentralization-swap mode.**
+
+After the SNS canister creation, the canisters exist but are not yet fully functional - the SNS is in **pre-decentralization-swap mode**.
+
+At this point, the SNS ledger only has two accounts: 
```

Comment:
FYI: This is not strictly true, there will also be staking accounts that back up the initial developer neurons. These are illiquid in the same manner that the below accounts are (governed by control & mode)

Reply:
Ah reading the following paragraph indicates there are tokens locked in neurons. Perhaps all this needs is a qualification for `only`.

Reply:
got it, will remove `only`